posix.mak

```diff
@@ -35,7 +36,7 @@ clean:
 test:
 	$(QUIET)$(MAKE) -C src -f posix.mak build-examples
 	$(QUIET)$(MAKE) -C src -f posix.mak unittest
-	$(QUIET)$(MAKE) -C test -f Makefile
+	$(DMD) -Itest -i -run test/run.d
```

Comment:
It fails on the auto tester as when `run.d` is running it tries to invoke `dmd`, but `dmd` is not in the path. Can you pass `$(DMD)`  as an environment variable or an argument to the command line, so it can be used from inside `run.d`?